dali/operators/reader/loader/coco_loader.cc

```diff
@@ -392,14 +504,14 @@ void CocoLoader::ParseJsonAnnotations() {
 
   for (auto &image_info : image_infos) {
     int objects_in_sample = 0;
-    std::vector<int> sample_rles_idx;
-    std::vector<std::string> sample_rles;
     int64_t polygons_sample_offset = polygon_data_.size();
```

Comment:
This reads as "offset of a sample in polygons". I think it means the opposite and then it should be sample_polygon(s)_offset, sample_polygon_count, sample_vertex_offset / vertices_offset, sample_vertex_count etc.
Offsets can be preceded by a plural form because it's the offset of a block/range of polygons - "polygons" is the object.
Count should be preceded by a singular form, because we're going into the aforementioned region and counting individual object. If you're terribly attached to a plural form, then it could be polygons_size / vertices_size.